app/src/main/java/com/dcrandroid/fragments/SendFragment.kt

```diff
@@ -309,53 +307,25 @@ class SendFragment : Fragment(), AdapterView.OnItemSelectedListener {
+    private fun showConfirmTransactionDialog() {
         if(context == null || activity == null){
             return
         }
-
         send_error_label.text = null
-        val dialogTitle = CoinFormat.format(
-                String.format(Locale.getDefault(), "%s %s DCR", getString(R.string.send), Utils.removeTrailingZeros(Mobilewallet.amountCoin(amount)))
-        )
-
-        val dialogBuilder = AlertDialog.Builder(context!!)
-        val inflater = activity!!.layoutInflater
-        val dialogView = inflater.inflate(R.layout.input_passphrase_box, null)
-        dialogBuilder.setCancelable(false)
-        dialogBuilder.setView(dialogView)
-
-        val passphrase = dialogView.findViewById<EditText>(R.id.passphrase_input)
+        val srcAccount = accountNumbers[send_account_spinner.selectedItemPosition]
 
-        dialogBuilder.setView(dialogView)
+        val unsignedTransaction = wallet.constructTransaction(send_dcr_address.text.toString(), amount, srcAccount, requiredConfirmations, isSendAll)
+        val transactionDialog = ConfirmTransactionDialog(context)
+                .setAddress(send_dcr_address.text.toString())
+                .setAmount(amount)
+                .setFee(unsignedTransaction.estimatedSignedSize)
+                .setPositiveButton(DialogInterface.OnClickListener { _, _ ->
+                    val pass = util!!.get(Constants.PASSPHRASE)
+                    startTransaction(pass)
+                    })
 
-        dialogBuilder.setTitle(dialogTitle)
-        dialogBuilder.setMessage(String.format(Locale.getDefault(),"%s %s DCR", getString(R.string.transaction_confirmation), Utils.formatDecred(amount)))
-
-        dialogBuilder.setPositiveButton(R.string.done) { _, _ ->
-            val pass = passphrase.text.toString()
-            if (pass.isNotEmpty()) {
-                val srcAccount = accountNumbers[send_account_spinner.selectedItemPosition]
-                try {
-                    val unsignedTransaction = wallet.constructTransaction(send_dcr_address.text.toString(), amount, srcAccount, requiredConfirmations, isSendAll)
-                    val transactionDialog = ConfirmTransactionDialog(context)
-                            .setAddress(send_dcr_address.text.toString())
-                            .setAmount(amount)
-                            .setFee(unsignedTransaction.estimatedSignedSize)
-                            .setPositiveButton(getString(R.string.send), DialogInterface.OnClickListener { _, _ ->
-                                startTransaction(pass)
-                            })
-                    transactionDialog.show()
-                } catch (e: Exception) {
-                    e.printStackTrace()
-                }
-            }
-        }
-        dialogBuilder.setNegativeButton(R.string.cancel, null)
 
-        val b = dialogBuilder.create()
-        b.show()
-        b.getButton(AlertDialog.BUTTON_POSITIVE).setTextColor(Color.BLUE)
+        transactionDialog.show()
```

Comment:
```suggestion
        transactionDialog.setCancelable(true)
        transactionDialog.show()
```

Make transaction dialog cancelable to fix #162

Reply:
> Suggested change Beta 
> [Give us feedback](https://www.research.net/r/SuggestedChanges)
>         transactionDialog.show()
>         transactionDialog.setCancelable(true)
>         transactionDialog.show()
> Apply suggestion 
> Make transaction dialog cancelable to fix #162

done in 7381bf9 commit